morphic.js

```diff
@@ -4496,6 +4496,56 @@ CursorMorph.prototype.init = function (aStringOrTextMorph) {
         this.target.setAlignmentToLeft();
     }
     this.gotoSlot(this.slot);
+    
+    // Add hidden text box for copying and pasting
+    this.hiddenText = document.createElement('textarea');
+    this.target.hiddenText = this.hiddenText;
```

Comment:
I actually don't see a reason for `hiddenText` to be in the target... I might've just forgotten to remove it after doing tests.